### ISSUE TITLE
Move Nrpm mover model to the Obsolete package

### ISSUE THREAD

**Issue by hcasperfu** [ibpsa/modelica-ibpsa#1704]
First discussed in https://github.com/ibpsa/modelica-ibpsa/pull/1646#issuecomment-1404315591.

This issue is to move `IBPSA.Fluid.Movers.SpeedControlled_Nrpm` (and `.Preconfigured.SpeedControlled_Nrpm`) to `IBPSA.Obsolete`.
Reasons behind this:
- About 33 lines of code are copied between `IBPSA.Fluid.Movers.{SpeedControlled_Nrpm,SpeedControlled_y}`. This makes the maintenance of these two models cumbersome and error-prone. (As a note, the duplication between the two flow-controlled models is about 13 lines.)
- Introducing another layer of inheritance to absorb these copied lines would probably make the `Movers` package even more complex and difficult to maintain for future developers.
- We (LBNL) have no knowledge of user actually using the `Nrpm` model.

For the reasons above, it makes sense to move the `Nrpm` mover model to the `Obsolete` package and get it out of the way.

I will leave this issue open for three weeks to leave time for feedback on this proposed change before working on the implementation.

**Comment by mwetter** [comment]
IDEAS is not using the `_Nrpm` models currently.
@cvering : Is AixLib using it?
@nytschgeusen : Is BuildingSystems using it?

**Comment by nytschgeusen** [comment]
BuildingSystems is not the using _Nrpm models

**Comment by FWuellhorst** [comment]
AixLib uses these models, but could switch to the `_y` version. In an internal discussion, a lot of confusion was raised with regard to the parameters and documentation of the `_y` model. That's why we always used the `nrpm` one:
- The input says "Constant normalized rotational speed", although it is not constant
- The parameter `speed_nominal=1` in `per` seems redundant and adds complexity.

**Comment by FelixStege** [comment]
Generally, I think it makes sense to switch to the `_y`. Especially if the `AixLib.Fluid.Movers.Data.Generic` record is adapted for this, as it currently still provides the parameters for both models. If I see it correctly, the difference between the two models is only the scaling of the setpoint signal. Does it make sense to specify an upper and lower limit for the setpoint signal, with which the signal is scaled from 0 to 1? In this way, both modeling approaches can be represented with one model

**Comment by mwetter** [comment]
@FWuellhorst @FelixStege : Thanks for the feedback from AixLib. I would rather not add scaling of the signal, as this will lead again to the complexity of the current implementation. The ultimate goal was to reduce this complexity. If `AixLib` users want to use a control signal that is in rpm, maybe best is to add a gain between controller and mover that normalizes the signal. 
I am not sure I understand the "upper and lower limit" as this is something we don't have in the model yet, and as movers don't have such a limiter, but the limiter is rather part of a controller, I would put a limiter into the control sequence if desired.

**Comment by hcasperfu** [comment]
Thanks to everyone for the discussion and feed back. I will start working on the implementation.

Besides moving the `Fluid.Movers.{SpeedControlled_Nrpm,Preconfigured.SpeedControlled_Nrpm}` to `Obsolete`, I will also replace the use of `_Nrpm` with `_y` in the following example/validation models:
- `Validation.{PowerSimplified,Pump_Nrpm_stratos,Pump_stratos,PumpCurveDerivatives,SpeedControlled_Nrpm}`

However, the following models use both the `_Nrpm` and `_y` movers:
- `Validation.BaseClasses.ControlledFlowMachine`
- `Examples.MoverParameter`

@mwetter What should I do with them? Do I just delete the `_Nrpm` bit?

**Comment by mwetter** [comment]
@hcasperfu : Yes, the `_Nrpm` should be deleted unless the examples/validations are in `IBPSA.Obsolete`. I suggest you copy the example/validation to `IBPSA.Obsolete`, and then in the original example/validation, delete the instances that use `_Nrpm`. Then, eventually we can delete the movers that takes `_Nrpm` as input from the `Obsolete` package (in a year or so).
